cli/cmds/cluster/create_cmd.go

```diff
@@ -0,0 +1,202 @@
+package cluster
```

Comment:
is there a reason to rewrite the file instead of changes to the create.go?

Reply:
if the reason because of the flags file, we can just leave the file as it is "create.go" and the flags file can be create_flags.go or flags.go

Reply:
Yeah I did a back-and-forth actually. I wasn't sure, and I left it like this because this way the file `create_cmd.go` was ordered before the `create_flags.go` lol. But I agree that is probably better to have it just as `create.go`. 😅